google/resumable_media/requests/download.py

```diff
@@ -19,6 +19,9 @@
 from google.resumable_media.requests import _helpers
 
 
+_SINGLE_GET_CHUNK_SIZE = 8192
```

Comment:
I'd love some guidance here on what makes sense, maybe @thobrla has thought about this?